Makefile

```diff
@@ -17,8 +17,12 @@ audit:
 lint:
 	exit
 
+.PHONY: fmt
+fmt:
+	go fmt ./...
+
 .PHONY: build
-build:
+build: fmt
```

Comment:
Please remove this - we don't want ci formatting code before building binary; the relationship between the code in build and in git repository should be exactly the same